.rubocop.yml

```diff
@@ -0,0 +1,42 @@
+# use the shared YaST defaults
+inherit_from:
+  /usr/share/YaST2/data/devtools/data/rubocop_yast_style.yml
+
+# Redundant returns add legibility for developers used to other languages
```

Comment:
still looks like copy from storage-ng rubocop. Other yast parts do not need to look like C++

Reply:
Yes, you are right. I took the storage-ng configuration as the base for this one. Fixed.